---
layout: page
title: Publications
description: Yuchao Jiang's publications
---

**For a complete list, see my [Google Scholar](https://scholar.google.com/citations?user=cVLhgc4AAAAJ&hl=en).**

&dagger; indicates co-first authors; &sect; indicates (co-)corresponding authors.

Tong CY, Li C, Hurni C, Jacq A, Nie X, Guy C, Suh JH, Wong R, Merlin C, Naef F, Menet J<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. Single-cell multiomic analysis of circadian rhythmicity in mouse liver, ***bioRxiv***, 2025. ([link](https://www.biorxiv.org/content/10.1101/2025.04.03.647044v1))

Coulter A, Tong CY, Ni Y<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. distQTL: distribution quantitative trait loci identification by population-scale single-cell data, ***bioRxiv***, 2025.

Gui L, **Jiang Y**, Wang J. Aggregating dependent signals with heavy-tailed combination tests. ***Biometrika***, to appear, 2025. ([link](https://arxiv.org/abs/2310.20460v2))

Weideman AK<sup>&sect;</sup>, Wang R, Ibrahim JG<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. Canopy2: tumor phylogeny inference by bulk DNA and single-cell RNA sequencing. ***Statistics in Biosciences***, 1-43, 2025. ([link](https://link.springer.com/article/10.1007/s12561-024-09466-1))

Kose C, Lindsey-Boltz LA, Sancar A<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. Genome-wide analysis of transcription-coupled repair reveals novel transcription events in Caenorhabditis elegans. ***PLOS Genetics***, 20 (7), e1011365, 2024. ([link](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1011365))

Manickam A<sup>&dagger;</sup>, Mei W<sup>&dagger;</sup>, Peterson JJ, Harigaya Y, Murdoch DM, Margolis DM, Oesterling A, Guo Z, Rudin CD, **Jiang Y<sup>&sect;</sup>**, Browne EP<sup>&sect;</sup>. Integrated single-cell multiomic analysis of HIV latency reversal reveals novel regulators of viral reactivation. ***Genomics, Proteomics & Bioinformatics***, 22 (1), qzae003, 2024. ([link](https://academic.oup.com/gpb/advance-article/doi/10.1093/gpbjnl/qzae003/7515255))


Mann B, Zhang X, Bell N, Adefolaju A, Thang M, Dasari R, Valdivia A, Quinsey C, Rauf Y, Cassidy N, Yang Y, Viziri C, Rego S, **Jiang Y**, Dunn D, Floyd S, Baldwin A, Hingtgen S, Satterlee A. A normalized ex vivo platform for functional precision diagnosis of patient brain tumor tissue. ***Cell Reports Medicine***, 4 (6), 101042, 2023. ([link](https://doi.org/10.1016/j.xcrm.2023.101042))


Agarwal A, Zhao F, **Jiang Y**, Chen L. TIVAN-indel: A computational framework for annotating and predicting noncoding regulatory small insertion and deletion. ***Bioinformatics***, 39 (2), btad060, 2023. ([link](https://academic.oup.com/bioinformatics/article/39/2/btad060/7008323))


Dong M, He Y, **Jiang Y**, Zou F. Joint gene network construction by single-cell RNA sequencing data. ***Biometrics***, 79 (2), 915-925, 2023. ([link](https://onlinelibrary.wiley.com/doi/10.1111/biom.13645))


Guan PY, Lee JS, Wang L, Lin KZ, Wen M, Chen L, **Jiang Y<sup>&sect;</sup>**. Destin2: integrative and cross-modality analysis of single-cell chromatin accessibility data. ***Frontiers in Genetics***, 14:1089936, 2023. ([link](https://www.frontiersin.org/articles/10.3389/fgene.2023.1089936/full))


**Jiang Y<sup>&sect;</sup>**, Harigaya Y, Zhang Z, Zhang H, Zang C, Zhang NR<sup>&sect;</sup>. Nonparametric single-cell multiomic characterization of three-way relationships between transcription factors, target genes, and chromatin accessibilities. ***Cell Systems***, 13 (9), 737-751, 2022. ([link](https://www.sciencedirect.com/science/article/abs/pii/S2405471222003489))


Wang R, Lin D<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. EPIC: inferring relevant cell types for complex traits by integrating genome-wide association studies and single-cell RNA sequencing. ***PLOS Genetics***, 18(6), e1010251, 2022. ([link](https://journals.plos.org/plosgenetics/article/authors?id=10.1371/journal.pgen.1010251))


Wang R, **Jiang Y<sup>&sect;</sup>**. Copy number variation detection by single-cell DNA sequencing with SCOPE. ***Methods in Molecular Biology***, 2493, 279-288, 2022. ([link](https://link.springer.com/protocol/10.1007/978-1-0716-2293-3_18))


Dong M, Thennavan A, Urrutia E, Li Y, Perou CM, Zou F<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. SCDC: bulk gene expression deconvolution by multiple single-cell RNA sequencing references. ***Briefings in Bioinformatics***, 22 (1), 416-427, 2021. ([link](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz166/5699815))


Mei W, Jiang Z, Chen L, Chen Y, Sancar A<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. Genome-wide circadian rhythm detection methods: systematic evaluations and practical guidelines. ***Briefings in Bioinformatics***, 22 (3), bbaa135, 2021. ([link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8138819/))


**Jiang Y<sup>&dagger;</sup><sup>&sect;</sup>**, Li W<sup>&dagger;</sup>, Lindsey-Boltz LA<sup>&dagger;</sup>, Yang Y, Li Y, Sancar A<sup>&sect;</sup>. Super hotspots and super coldspots in the repair of UV-induced DNA damage in the human genome. ***Journal of Biological Chemistry***, 296, 100581, 2021. ([link](https://linkinghub.elsevier.com/retrieve/pii/S0021-9258(21)00361-6))


Liu Z, Ruter DL, Quigley K, **Jiang Y**, Bautch VL. Single-cell RNA sequencing reveals endothelial cell gene expression heterogeneity under homeostatic shear stress. ***Arteriosclerosis, Thrombosis, and Vascular Biology***, 41 (10), 2575-2584, 2021. ([link](https://www.ahajournals.org/doi/full/10.1161/ATVBAHA.121.316797))


Wang Y, **Jiang Y**, Yao B, Huang K, Liu Y, Wang Y, Qin X, Chen L. WEVar: a novel supervised learning framework for predicting functional noncoding variants. ***Briefings in Bioinformatics***, 22 (6), bbab189, 2021. ([link](https://academic.oup.com/bib/article-abstract/22/6/bbab189/6279833?redirectedFrom=fulltext))


Wang Y, Bhattacharya T, **Jiang Y**, Qin X, Wang Y, Liu Y, Saykin AJ, Chen L. A novel deep learning method for predictive modeling of microbiome data. ***Briefings in Bioinformatics***, 22 (3), bbaa073, 2021. ([link](https://academic.oup.com/bib/article/22/3/bbaa073/5835556))


Wang R, Lin D, **Jiang Y<sup>&sect;</sup>**. SCOPE: a normalization and copy number estimation method for single-cell DNA sequencing. ***Cell Systems***, 10 (5), 445-452, 2020. ([link](https://doi.org/10.1016/j.cels.2020.03.005))


Huh R, Yang Y, **Jiang Y**, Shen Y, Li Y. SAME-clustering: single-cell aggregated clustering via mixture model ensemble. ***Nucleic Acids Research***, 48 (1), 86-95, 2020. ([link](https://academic.oup.com/nar/article/48/1/86/5644992))


Yimit A, Adebali O, Sancar A<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. Differential damage and repair of DNA-adducts induced by anti-cancer drug cisplatin across mouse organs. ***Nature Communications***, 10 (1), 309, 2019. ([link](https://www.nature.com/articles/s41467-019-08290-2))


Urrutia E, Chen L, Zhou H, **Jiang Y<sup>&sect;</sup>**. Destin: toolkit for single-cell analysis of chromatin accessibility. ***Bioinformatics***, 35 (19), 3818-3820, 2019. ([link](https://academic.oup.com/bioinformatics/article/35/19/3818/5367832/))


Yang Y, Hu J, Selby CP, Li W, Yimit A, **Jiang Y<sup>&sect;</sup>**, Sancar A<sup>&sect;</sup>. Single nucleotide resolution analysis of nucleotide excision repair of ribosomal DNA in humans and mice. ***Journal of Biological Chemistry***, 294 (1), 210-217, 2019. ([link](https://www.jbc.org/article/S0021-9258(20)36891-5/fulltext))


Li W, Liu W, Kakoki A, Wang R, Adebali O, **Jiang Y**, Sancar A. Nucleotide excision repair capacity increases during differentiation of human embryonic carcinoma cells into neurons and muscle cells. ***Journal of Biological Chemistry***, 294 (15), 5914-5922, 2019. ([link](https://www.jbc.org/article/S0021-9258(20)36664-3/fulltext))


Dong M, **Jiang Y<sup>&sect;</sup>**. Single-cell allele-specific gene expression analysis. ***Methods in Molecular Biology***, 1935, 155-174, 2019. ([link](https://link.springer.com/protocol/10.1007/978-1-4939-9057-3_11))


**Jiang Y<sup>&sect;</sup>**, Wang R, Urrutia E, Anastopoulos IN, Nathanson KL, Zhang NR<sup>&sect;</sup>. CODEX2: full-spectrum copy number variation detection by high-throughput DNA sequencing. ***Genome Biology***, 19 (1), 202, 2018. ([link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1578-y))


Urrutia E, Chen H, Zhou Z, Zhang NR<sup>&sect;</sup>, **Jiang Y<sup>&sect;</sup>**. Integrative pipeline for profiling DNA copy number and inferring tumor phylogeny. ***Bioinformatics***, 34 (12), 2126-2128, 2018. ([link](https://doi.org/10.1093/bioinformatics/bty057))


**Jiang Y**, Zhang NR<sup>&sect;</sup>, Li M<sup>&sect;</sup>. SCALE: modeling allele-specific gene expression by single-cell RNA sequencing. ***Genome Biology***, 18 (1), 74, 2017. ([link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1200-8))


Chen H, **Jiang Y**, Maxwell KN, Nathanson KL, Zhang NR. Allele-specific copy number profiling using whole exome sequencing. ***Annals of Applied Statistics***, 11 (2), 1169-1192, 2017. ([link](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-11/issue-2/Allele-specific-copy-number-estimation-by-whole-exome-sequencing/10.1214/17-AOAS1043.full)) 


Garman B, Anastopoulos IN, Krepler C, Brafford P, Sproesser K, **Jiang Y**, Wubbenhorst B, Amaravadi R, Bennett J, Beqiri M, Elder D, Flaherty KT, Frederick DT, Gangadhar TC, Guarino M, Hoon D, Karakousis G, Liu Q, Mitra N, Petrelli NJ, Schuchter L, Shannan B, Shields CL, Wargo J, Wenz B, Wilson MA, Xiao M, Xu W, Xu X, Yin X, Zhang NR, Davies MA, Herlyn M, Nathanson KL. Genetic and Genomic Characterization of 462 Melanoma Patient-Derived Xenografts, Tumor Biopsies, and Cell Lines. ***Cell Reports***, 21 (7), 1936-1952, 2017. ([link](https://www.cell.com/cell-reports/fulltext/S2211-1247(17)31499-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124717314997%3Fshowall%3Dtrue))


Maxwell KN, Wubbenhorst B, Wenz BM, Sloover DD, Pluta J, Emery L, Barrett A, Kraya A, Anastopoulos IN, Yu S, **Jiang, Y**, Chen H, Zhang NR, Hackman N, D’Andrea K, Daber R, Morrissette JJD, Mitra N, Feldman M, Domchek SM, Nathanson KL. BRCA locus-specific loss of heterozygosity in germline BRCA1 and BRCA2 carriers. ***Nature Communications***, 8 (1), 319, 2017. ([link](https://www.nature.com/articles/s41467-017-00388-9))


**Jiang Y**, Qiu Y, Minn AJ, Zhang NR. Assessing tumor heterogeneity and tracking longitudinal and spatial clonal evolution by next-generation sequencing. ***Proceedings of the National Academy of Sciences***, 113 (37), E5528-E5537, 2016. ([link](http://www.pnas.org/content/113/37/E5528), [news](http://www.uphs.upenn.edu/news/News_Releases/2016/09/jiang/))


**Jiang Y**, Oldridge DA, Diskin SJ, Zhang NR. CODEX: a normalization and copy number variation detection method for whole exome sequencing. ***Nucleic Acids Research***, 43 (6), e39-e39, 2015. ([link](https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gku1363))
