---
layout: page
title: Software
description: Yuchao Jiang's software
---

### Cancer Genomics and Intratumor Heterogeneity

**Canopy**: Assessing intratumor heterogeneity and tracking longitudinal and spatial clonal evolutionary history by next-generation sequencing ([CRAN](https://cran.r-project.org/web/packages/Canopy/index.html), [GitHub](https://github.com/yuchaojiang/Canopy), [vignettes](https://cran.rstudio.com/web/packages/Canopy/vignettes/Canopy_vignettes.pdf))

**MARATHON**: Integrative pipeline for profiling DNA copy number and inferring tumor phylogeny ([GitHub](https://github.com/yuchaojiang/MARATHON), [vignettes](https://rawgit.com/yuchaojiang/MARATHON/master/notebook/MARATHON.html))

**Canopy2**: Tumor phylogeny inference using bulk DNA and single-cell RNA sequencing ([GitHub](https://github.com/annweideman/canopy2/), [vignettes](https://htmlpreview.github.io/?https://github.com/annweideman/canopy2/blob/main/vignettes/canopy2.html))

### Copy Number Variation & Intratumor Heterogeneity

**CODEX**: A normalization and copy number variation detection method for whole exome sequencing ([Bioconductor](http://www.bioconductor.org/packages/release/bioc/html/CODEX.html), [GitHub](https://github.com/yuchaojiang/CODEX), [vignettes](http://www.bioconductor.org/packages/release/bioc/vignettes/CODEX/inst/doc/CODEX_vignettes.pdf))

**CODEX2**: Full-spectrum copy number variation detection by high-throughput DNA sequencing ([GitHub](https://github.com/yuchaojiang/CODEX2), [vignettes](http://htmlpreview.github.io/?https://github.com/yuchaojiang/CODEX2/blob/master/demo/CODEX2.html))

**Falcon-X**: Allele-specific copy number estimation by whole-exome sequencing ([CRAN](https://CRAN.R-project.org/package=falconx))

### Single-Cell Omics

**SCALE**: Modeling allele-specific gene expression by single-cell RNA sequencing ([GitHub](https://github.com/yuchaojiang/SCALE), [vignettes](https://github.com/yuchaojiang/SCALE/blob/master/demo/SCALE_vignettes.pdf))

**Destin**: Toolkit for single-cell analysis of chromatin accessibility ([GitHub](https://github.com/urrutiag/destin), [vignettes1](https://rawgit.com/urrutiag/destin/master/package/vignettes/destinBuenrostroMouse.html), [vignettes2](https://rawgit.com/urrutiag/destin/master/package/vignettes/destinPreisslP56.html))

**Destin2**: Integrative and cross-modality analysis of single-cell chromatin accessibility data ([GitHub](https://github.com/yuchaojiang/Destin2), [vignettes1](http://htmlpreview.github.io/?https://github.com/yuchaojiang/Destin2/blob/main/package/vignettes/destin2vignette_pbmc.html), [vignettes2](http://htmlpreview.github.io/?https://github.com/yuchaojiang/Destin2/blob/main/package/vignettes/destin2vignette_brain.html))

**SCDC**: Deconvolution of bulk gene expression by single-cell RNA sequencing data ([GitHub](https://meichendong.github.io/SCDC), [vignettes](https://meichendong.github.io/SCDC/articles/SCDC.html))

**SCOPE**: A normalization and copy number estimation method for single-cell DNA sequencing ([Bioconductor](https://bioconductor.org/packages/SCOPE/), [GitHub](https://github.com/rujinwang/SCOPE), [vignettes](http://bioconductor.org/packages/devel/bioc/vignettes/SCOPE/inst/doc/SCOPE_vignette.html))

**TRIPOD**: Nonparametric interrogation of transcriptional regulation in single-cell RNA and chromatin accessibility multiomic data ([GitHub](https://github.com/yuchaojiang/TRIPOD), [vignettes](http://htmlpreview.github.io/?https://github.com/yuchaojiang/TRIPOD/blob/main/vignettes/TRIPOD.html))

**EPIC**: Inferring relevant tissues and cell types for complex traits by integrating genome-wide association studies and single-cell RNA sequencing ([GitHub](https://github.com/rujinwang/EPIC), [vignettes](http://htmlpreview.github.io/?https://github.com/rujinwang/EPIC/blob/master/vignettes/EPIC_vignette.html))

**distQTL**: Distribution quantitative trait loci identification by population-scale single-cell data ([GitHub](https://github.com/alexandercoulter/distQTL), [vignettes](https://alexandercoulter.github.io/distQTL/intro-distQTL.html))

### DNA Damage, Repair, and Circadian Rhythm

**Benchmarking methods for circadian rhythm detection**: [vignettes](https://htmlpreview.github.io/?https://github.com/wenwenm183/Circadian_Genes_Benchmark/blob/master/vignettes/Vignettes-of-Circadian-Paper.html)

**DNA damage and excision repair**: [GitHub](https://github.com/yuchaojiang/damage_repair)

**Single-cell multiomic analysis of circadian rhythmicity**: [GitHub](https://github.com/yuchaojiang/circadian_sc_multiomics)
